_includes/master/reqs-sys.md

```diff
@@ -20,13 +20,26 @@
 ## Key/value store
 
 {{site.prodname}} {{page.version}} requires a key/value store accessible by all
-{{site.prodname}} components. {% if include.orch == "Kubernetes" %} On Kubernetes,
-you can configure {{site.prodname}} to access an etcdv3 cluster directly or to
-use the Kubernetes API datastore.{% endif %}{% if include.orch == "OpenShift" %} On
-OpenShift, {{site.prodname}} can share an etcdv3 cluster with OpenShift, or
-you can set up an etcdv3 cluster dedicated to {{site.prodname}}.{% endif %}
+{{site.prodname}} components. {% if include.orch == "Kubernetes" %} You can choose either
+of the following options.
+
+- Use the Kubernetes API datastore.
+
+- Configure {{site.prodname}} to access an etcdv3 cluster directly (required if you plan to protect
```

Comment:
I think this is too much information - I don't think we should call out bare-metal hosts or the quickstart guide from here.

Reply:
That's fine, but I'm just not currently convinced the requirements doc is the right place for them.

We can chat it over if you disagree!